<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPF</class>
 <widget class="QWidget" name="DPF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPF</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Threshold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Weight</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="DPF_outputMap_cb">
        <property name="text">
         <string>Generate Output Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="DPF_weight_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="DPF_inputMap_cb">
        <property name="text">
         <string>Use Input Map</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="DPF_apply_btn">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="DPF_threshold_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="DPF_HWDetect_cb">
        <property name="text">
         <string>HW Detection</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="5">
       <widget class="QLabel" name="DPF_inputMapinfo_l">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Select Confidence Range</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="DPF_s_lb">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select S</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="5">
       <widget class="QTableWidget" name="DPF_inputMap_tbl"/>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="DPF_load_btn">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="DPF_minConfidence_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSpinBox" name="DPF_maxConfidence_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="5">
       <widget class="QProgressBar" name="DPF_progress_pb">
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Statistics</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Fixed Pixels</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="DPF_fixedPixels_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Written to map</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="DPF_nOutCorrections_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dropped from writing</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="DPF_ndroppedCorrections_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DPF_HWDetect_cb</tabstop>
  <tabstop>DPF_threshold_sb</tabstop>
  <tabstop>DPF_weight_sb</tabstop>
  <tabstop>DPF_outputMap_cb</tabstop>
  <tabstop>DPF_inputMap_cb</tabstop>
  <tabstop>DPF_load_btn</tabstop>
  <tabstop>DPF_apply_btn</tabstop>
  <tabstop>DPF_minConfidence_sb</tabstop>
  <tabstop>DPF_maxConfidence_sb</tabstop>
  <tabstop>DPF_s_lb</tabstop>
  <tabstop>DPF_inputMap_tbl</tabstop>
  <tabstop>DPF_fixedPixels_sb</tabstop>
  <tabstop>DPF_nOutCorrections_sb</tabstop>
  <tabstop>DPF_ndroppedCorrections_sb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
